.gitignore

```diff
@@ -102,3 +102,4 @@ target/
 
 # Pycharm
 .idea/
+docs/source/_build/
```

Comment:
Or, there is also a section for "Sphinx documentation" ... under that might also be desirable 

(but the existence of a exclusion for Sphinx already that is very similar to the exclusion we are adding makes me wonder if we shouldn't adhere to the existing convention...)